src/configuration/useAuthorityLinks.ts

```diff
@@ -0,0 +1,39 @@
+import links from "./links.json"
+
+const DEFAULT_LOCALE = "en"
+type TranslationValue = Record<string, string>
+type LinkData = {
+  url: string
+  label: TranslationValue
+}
+type LinkKeys = "about" | "legal"
+type AuthorityLinksContent = Record<LinkKeys, LinkData[]>
+type DisplayableLink = {
+  url: string
+  label: string
+}
+
+const useAuthorityLinks = (
```

Comment:
This code looks great! but i wonder if we can simply a bit. for example, do we need to check the label here? could we just pass in an array of linkData? and do the check of if the config provides a sensible list in the component?

Reply:
Wouldn't this makes us have the same behavior scattered across multiple components? Would it be our responsibility to exercise that behavior in all those places instead of having the encapsulation here?